### PR TITLE
Add labels,kubelet and bastion_host_key to Terraform Integration reference

### PR DESCRIPTION
**Pull request by akurz**
Signed-off-by: Andreas Kurz <andreas.kurz@gmail.com>

### CODE REVIEW

**Comment by embik** [comment]
@xmudrii can you take a look at this PR please?

File: content/kubeone/v1.5/references/terraform-integration/_index.en.md [hunk 1]
```diff
@@ -99,6 +99,16 @@ output "kubeone_hosts" {
       bastion              = aws_instance.bastion.public_ip
       bastion_port         = var.bastion_port
       bastion_user         = var.bastion_user
+      bastion_host_key     = var.bastion_host_key
```
Comment (xmudrii) [inline content/kubeone/v1.5/references/terraform-integration/_index.en.md:102]: This feature is available only starting with KubeOne 1.6 (to be released in February). Please drop this line from this file.

File: content/kubeone/v1.5/references/terraform-integration/_index.en.md [hunk 2]
```diff
@@ -216,6 +235,16 @@ output "kubeone_static_workers" {
...
```
Comment (xmudrii) [inline content/kubeone/v1.5/references/terraform-integration/_index.en.md:238]: This feature is available only starting with KubeOne 1.6 (to be released in February). Please drop this line from this file.

**Review by xmudrii — CHANGES_REQUESTED** [review]

File: content/kubeone/v1.5/references/terraform-integration/_index.en.md (hunk 1 above)
Comment (akurz) [reply content/kubeone/v1.5/references/terraform-integration/_index.en.md]: Dropped, thanks!

File: content/kubeone/v1.5/references/terraform-integration/_index.en.md (hunk 2 above)
Comment (akurz) [reply content/kubeone/v1.5/references/terraform-integration/_index.en.md]: Thanks, done.

**Comment by akurz** [comment]
Hi @xmudrii - I adopted the PR as you requested. Do you think this is fine for merging? 

thx & Cheers,
Andreas

**Comment by akurz** [comment]
Hi, looks like I messed up with git - will close this PR and create a new one rebased on current master.

**Pull request closed without merging**